libs/blocks/global-footer/global-footer.js

```diff
@@ -28,9 +31,18 @@ const CONFIG = {
 };
 
 class Footer {
-  constructor(footerEl, contentUrl) {
-    this.footerEl = footerEl;
-    this.contentUrl = contentUrl;
+  /**
```

Comment:
Yeah, I was torn too, I wanted to avoid too many `this.prop = config.prop` declarations, but in the end we add JSDoc to make things clear, so it's not necessarily the best option. I'll probably revert to `this.prop = config.prop` to keep track of what config is actually used 🤷